spec/index.html

```diff
@@ -1206,6 +1231,10 @@ <h2>RDFS Interpretations</h2>
       x is in ICEXT(I(<code>rdfs:Datatype</code>)) then <span >&lt; x,
       I(<code>rdfs:Literal</code>) &gt; is in IEXT(I(<code>rdfs:subClassOf</code>))</span></p></td>
     </tr>
+    <tr>
+    <td class="semantictable" id="rdfssemcond11"><p>If
+      exist x,y,z such that RE(x,z,y)=r, or exists x such that &lt; x,r &gt; is in IEXT([I+A](<code>rdf:reifies</code>)), <br/> then &lt; r,[I+A](<code>rdfs:Proposition</code>)&gt; is in IEXT([I+A](<code>rdf:type</code>))</p></td>
```

Comment:
Line lengths, again. I'm not changing it, but I think `exist` in the first line below should be `exists`. And, this is the rare case where `<br/>` is preferred over `<br/><br/>`.
```suggestion
      exist x,y,z such that RE(x,z,y)=r, or 
      exists x such that &lt; x,r &gt;
         is in IEXT([I+A](<code>rdf:reifies</code>)),
      <br/>
      then &lt; r,[I+A](<code>rdfs:Proposition</code>)&gt;
         is in IEXT([I+A](<code>rdf:type</code>))</p></td>
```

Reply:
Fixed and committed.